siac/cmds.go

```diff
@@ -47,20 +48,64 @@ func updatecmd(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	fmt.Println(getResponse("/update", &url.Values{
-		"action": {args[0]},
-	}))
+	switch args[0] {
+	case "check":
+		if m, err := getResponse("/update/check", nil); err == nil {
+			if m["Available"].(bool) {
+				fmt.Printf("Update %s is available!\nRun 'siac update %s' to install it.\n", m["Version"], m["Version"])
+			} else {
+				fmt.Println("Up to date!")
+			}
+		} else {
+			fmt.Println("Update check failed:", err)
+		}
```

Comment:
trying to open a discussion on this so that we can make a standard. I'm not certain that using control statements instead of `else` is best, but I like it this way because you usually type `return` after detecting an error, so I want to standardize around always having a control statement after detecting errors, at least where possible. And then adding documentation when things don't flow like that.

The other thing is that having a bunch of strict conventions makes it easier to document stuff. If you find a good reason to break convention, you have to document why which makes everyone's life easier, because it's something that would normally otherwise go undocumented.

It's analogous to documenting everything, because the convention serves as documentation that doesn't need to be written.